src/Extensions/ElementalAreasExtension.php

```diff
@@ -81,7 +81,11 @@ public function getElementalTypes()
             $availableClasses = ClassInfo::subclassesFor(BaseElement::class);
         }
 
-        $disallowedElements = (array) $config->get('disallowed_elements');
+        if ($config->get('stop_element_inheritance')) {
```

Comment:
I think it'd be nice to use a positive form of config rather than a negative, e.g. `inherit_allowed_elements: true` by default and disable it to prevent. Thoughts?

Reply:
I've just kept with the existing config that's set so as not to cause any issues where that's already being used. To me it doesn't feel like it needs another config variable as these should do the same thing.

Reply:
Sorry, I thought you were adding a new config prop - all good!